core/app/models/spree/stock/prioritizer.rb

```diff
@@ -17,22 +17,32 @@ def prioritized_packages
       end
 
       private
-      def adjust_packages
-        inventory_units.each do |inventory_unit|
-          adjuster = @adjuster_class.new(inventory_unit, :on_hand)
-
-          visit_packages(adjuster)
 
-          adjuster.status = :backordered
-          visit_packages(adjuster)
+      def adjust_packages
+        packages.each do |package|
+          package.contents.each do |item|
+            adjuster = find_adjuster(item)
+
+            if adjuster.nil?
+              adjuster = build_adjuster(item, package)
+
+            elsif item.state == :on_hand && adjuster.status == :backordered
+              # Adjust old package content if it was backordered and reassign package and state
```

Comment:
Could you please elaborate this comment? Why package is replaced by adjuster package, and adjuster package by package?